### DISCUSSION TITLE
Any documentation around the GameTestHelper and how to use it with the current implementation?

### DISCUSSION THREAD

**Discussion by meza** [FabricMC/fabric-api#2461] [Mod Dev Support]
I'm trying to wrap my head around the GameTest ecosystem and am running into a major lack of documentation. Could anyone point me in the right direction? Examples could also help a lot.

**Comment by warjort** [comment]
I am not aware of any docs or tutorials. Maybe somebody else knows?

The overview of the idea is on the original pull request: https://github.com/FabricMC/fabric/pull/1622

As for examples, this is what github shows: https://github.com/search?q=fabricgametest&type=code
That interface is optional so there are likely other examples?
